ressources/modeles/modale_publication.html

```diff
@@ -0,0 +1,79 @@
+<!-- Le modal -->
+<div id="my-modal" class="modal fade">
+    <div class="modal-dialog">
+        <!-- Contenu du modal -->
+        <div id="publication-content" class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">Publication</h4>
+                <button type="button" class="btn-close" data-bs-dismiss="modal">&times;</button>
+                <!-- symbole x pour fermer -->
+            </div>
+
+            <div class="modal-body" id="page-publication">
+
+                <h2>Ajouter un nouveau cocktail à nos archives</h2>
+
+                <form>
+                    <label for="nom-cocktail">Nom du cocktail:</label><br>
+                    <textarea id="name-texte" name="nom-cocktail" rows="1" cols="50"></textarea><br>
+
+                    <label for="ingredient-cocktail">Ingrédients:</label><br>
+                    <div id="ingredient-input">
+                        <div id="add-form">
+                            <input type="text" id="ingredient-nom" placeholder="Nom de l'ingrédient...">
+                            <input type="text" id="ingredient-quantite" placeholder="Quantité...">
+                            <input type="text" id="ingredient-unit" placeholder="Unité...">
+
+                            <button class="button" id="ajouter-ingredient">
+                                <img class="btn-icon" src="../ressources/images/plus.svg" alt="+">
```

Comment:
L'attribut `alt` doit décrire l'objectif ou la fonction du bouton plutôt que l'image elle-même.
```suggestion
                                <img class="btn-icon" src="../ressources/images/plus.svg" alt="Ajouter un ingrédient">
```